src/collegetown.py

```diff
@@ -0,0 +1,35 @@
+from os import environ
+from time import sleep
+
+from yelpapi import YelpAPI
+
+from src.constants import (
+    YELP_LATITUDE,
+    YELP_LONGITUDE,
+    YELP_RADIUS,
+    YELP_NUMBER_LIMIT,
+    YELP_QUERY_DELAY,
+)
+
+API_KEY = environ.get('YELP_API_KEY')
+
+yelp_api = YelpAPI(API_KEY)
+
+def collegetown_search():
+  try:
+    search = yelp_api.search_query(
+        longitude=YELP_LONGITUDE,
+        latitude=YELP_LATITUDE,
+        radius=YELP_RADIUS,
+        limit=YELP_NUMBER_LIMIT
+    )
+    eateries = []
+    for place in search['businesses']:
+      eatery_id = place['id']
+      id_query = yelp_api.business_query(id=eatery_id)
+      eateries.append(id_query)
+      sleep(YELP_QUERY_DELAY)  # delay between query calls to not trigger yelp query limits
```

Comment:
clever, but you should never call `sleep` in a function because then you won't be able to handle more client requests, is there a better way to do this?

Reply:
Clients won't be calling this function at all.  Only the start_update() that gets called once a day